src/main/java/by/itechart/api/controller/UserController.java

```diff
@@ -23,16 +25,20 @@
 
     private final UserService userService;
 
+    @ApiPageable
```

Comment:
Other API documentation-related annotations placed in `UserControllerInfo`. Please, be consistent with placing annotations